cmd/src/bin/tikv-ctl.rs

```diff
@@ -1187,10 +1268,24 @@ fn main() {
                     SubCommand::with_name("region")
                         .about("print region info")
                         .arg(
-                            Arg::with_name("region")
-                                .short("r")
+                            Arg::with_name("regions")
```

Comment:
better not to change the existed command

Reply:
fixed